eth_typing/abi.py

```diff
@@ -170,7 +140,35 @@ class ABIFunctionInfo(TypedDict, total=False):
     """Function input parameters."""
 
 
-ABIElement = Union[ABIFunction, ABIConstructor, ABIFallback, ABIReceive, ABIEvent]
+class ABIErrorParam(TypedDict, total=False):
```

Comment:
Well, sorry to go back to the original thread but if this exact class can be used somewhere else, with the exact same properties, I think we should make this even more generic. `ABIParam` or `ABIComponent` (since this model fits into the `components` field). And then we can make use of it both in the `ABIError` and in the `ABIFunction` and anywhere else.

Reply:
Agh that's my bad, I will get that fixed.